content/docs/cucumber/api.md

```diff
@@ -848,6 +848,22 @@ The **Command-Line Interface Runner (CLI Runner)** is an executable Java class t
 ```
 java io.cucumber.core.cli.Main
 ```
+Note that you will need to add the following jars to your classpath, in addition to the location of your compiled .class files:
+
+* `cucumber-core-4.8.0.jar`
+* `gherkin-5.1.0.jar`
+* `cucumber-expressions-7.0.2.jar`
+* `datatable-1.1.14.jar`
+* `datatable-dependencies-1.1.14.jar`
+* `cucumber-java-4.8.0.jar`
+
+You will also need to provide the CLI with your step definitions via the `--glue` option followed by its package name, and the location of your feature file(s).
+
+For example:
+```shell
+java -cp "/home/annabaker/.m2/repository/io/cucumber/cucumber-core/4.8.0/cucumber-core-4.8.0.jar:/home/annabaker/.m2/repository/io/cucumber/gherkin/5.1.0/gherkin-5.1.0.jar:/home/annabaker/.m2/repository/io/cucumber/cucumber-expressions/7.0.2/cucumber-expressions-7.0.2.jar:/home/annabaker/.m2/repository/io/cucumber/datatable/1.1.14/datatable-1.1.14.jar:/home/annabaker/.m2/repository/io/cucumber/datatable-dependencies/1.1.14/datatable-dependencies-1.1.14.jar:/home/annabaker/.m2/repository/io/cucumber/cucumber-java/4.8.0/cucumber-java-4.8.0.jar:/home/annabaker/Development/git/hellocucumber/target/test-classes" io.cucumber.core.cli.Main ~/Development/git/hellocucumber/src/test/resources/hellocucumber/is_it_friday_yet.feature --glue hellocucumber
```

Comment:
Do you want to have your name here? (Not sure what, if anything, we use in other examples)

Reply:
I've since modified this.  It's much more readable now, what do you think?